build.rs

```diff
@@ -201,6 +201,8 @@ fn ignore(testsuite: &str, testname: &str, strategy: &str) -> bool {
             ("reference_types", _) if cfg!(feature = "old-x86-backend") => return true,
             // No simd support yet for s390x.
             ("simd", _) if platform_is_s390x() => return true,
+            // No memory64 support yet for s390x.
+            ("memory64", _) if platform_is_s390x() => return true,
```

Comment:
How come memory64 isn't implemented on s390x? Does it lower to unimplemented clif things at the moment?

Reply:
Yes.   This is all fixable, but I'd prefer to do that separately.